include/flash_map.h

```diff
@@ -33,9 +33,6 @@ extern "C" {
 #include <stddef.h>
 #include <sys/types.h>
 
-#define SOC_FLASH_0_ID 0	/* device_id for SoC flash memory driver */
```

Comment:
Here is the reason why this array is defined
In order to be compatible with the old name